opencti-platform/opencti-front/vite.config.mts

```diff
@@ -146,8 +146,7 @@ const depsToOptimize = [
   "@mui/lab/LoadingButton",
   "@mui/material/Breadcrumbs",
   "classnames",
-  "react-draggable",
-  "react-beautiful-dnd"
+  "react-draggable"
```

Comment:
Seems unlinked to task

Reply:
Sorry I forget to add explanation on that.
When I launch the front-end I had this issue: 
Failed to resolve dependency: react-beautiful-dnd, present in 'optimizeDeps.include'

So I remove this dependency from here, because it's not in the package.json anymore.